src/modules/baseSpec.ts

```diff
@@ -1,5 +1,4 @@
-/* eslint-env jasmine */
```

Comment:
Phew, I forgot how extensive these tests were and how much of a PITA it must have been to convert this over to TS.  Thanks!